docs/reference/announcements/870.md

```diff
@@ -0,0 +1,140 @@
+Learn more about these updates in Upcoming API Changes in Camunda 8.
+::: -->
+
+## Identity management updates <span class="badge badge--long" title="This feature affects SaaS">SaaS</span><span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
+
+The [Identity service](/self-managed/identity/what-is-identity.md) is enhanced to deliver greater flexibility, control, and security for both Self-Managed and SaaS users. These updates are part of our broader effort to streamline the platform’s architecture.
+
+### Cluster-level identity management
+
+Identity settings will be configured at the orchestration cluster level, allowing each cluster to have unique OIDC configurations. This cluster-specific setup empowers organizations to assign different identity providers (IdPs) across clusters, offering improved control over permissions and user group mappings, resulting in a more streamlined and efficient configuration experience.
+
+For SaaS customers, identity management in Camunda 8.7 remains consistent with Camunda 8.6, allowing the attachment of a single IdP per organization. However, cluster-level identity capabilities are provided for SaaS as well as Self-Managed. This means that user groups, roles, and access permissions can now be managed at the cluster level, giving SaaS customers the same granular access control as in Self-Managed environments.
+
+### Decoupling from Keycloak <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
+
+Built-in Keycloak integration in Self-Managed is removed, allowing customers to use any compatible IdP.
+
+- Keycloak remains fully supported as an external option.
+- OpenID Connect (OIDC) remains the standard for seamless integration with chosen IdPs.
+
+### Resource-based permissions
+
+Resource-level permissions are introduced for process definitions and web applications.
+
+- Admin users retain full access, but regular users must be granted specific permissions to perform actions/view resources.
+- For organizations that build custom front-ends and access Camunda via API, users with API permissions can still access process data through the V2 API.
+
+<!-- :::info
+Learn more about these updates in Introducing Enhanced Identity Management in Camunda 8.7.
+::: -->
+
+## Installation and deployment updates <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
+
+Camunda 8.7 introduces a streamlined architecture, consolidating core components such as Zeebe, Operate, and Tasklist into a single deployable unit. Enhanced deployment options are also included, such as new Kubernetes Helm guides, deployment reference architectures, and improved support for professional developers with Camunda 8 Run.
+
+You can download the alpha release of the unified package from the Camunda GitHub repository, either as an executable Java application (Camunda Orchestration Core) or a Docker image.
+
+### Helm charts
```

Comment:
@hamza-m-masood / @aabouzaid should we add the removal of the separate ingress here already? Its already merged and will be released with alpha2, right?

Reply:
@hisImminence Added :heavy_check_mark:

Reply:
Thank you!